src/momento/internal/_utilities/_momento_version.py

```diff
@@ -0,0 +1,14 @@
+momento_version = ""
+
+try:
+    # For python < 3.8
+    import importlib_metadata
+
+    momento_version = importlib_metadata.Distribution.from_name("momento").version  # type: ignore[no-untyped-call,misc]
+except ImportError:
+    # For python >= 3.8
+    from importlib.metadata import version  # type: ignore[import]
+
+    momento_version = version("momento")
+
+assert momento_version != ""
```

Comment:
Hmm this assert makes me bit nervous maybe better fail open here default to something like "undefined" instead it's just for agent header right? If you feel good with hard fail here though I can get behind it.

Reply:
I will move this to a unit test

Reply:
Done in [3a85dfb](https://github.com/momentohq/client-sdk-python/pull/343/commits/3a85dfb11ede1623439a27d6f8e4c174754cb2b9)